.prettierignore

```diff
@@ -1,3 +1,6 @@
+app/routes/entry.server.tsx
```

Comment:
We don't want to ignore anything in `app/`.  All of our code should be formatted.